packages/models/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@quality-metrics/models",
+  "version": "0.0.1",
+  "dependencies": {
+    "bundle-require": "^4.0.1"
```

Comment:
The dependencies are inaccurate, `bundle-require` isn't used here, but `zod` is for example.

Reply:
I guess we can use the build process to generate the deps automatically... Let me try.

Reply:
Hmmm the [linting rule](https://nx.dev/packages/eslint-plugin/documents/dependency-checks) does not work on my end...

Reply:
Yeah, the Nx rule should catch it, I believe. But it has to be in each project's `.eslintrc.json` file, so the fix should be to add that rule to `models` and `utils` projects (Nx only generates it there when you set it to publishable).